Traceback (most recent call last):
  File "/dtu/blackhole/1b/209339/text-to-image-generation-in-the-medical-domain/src/train_text_to_image_lora.py", line 978, in <module>
    main()
  File "/dtu/blackhole/1b/209339/text-to-image-generation-in-the-medical-domain/src/train_text_to_image_lora.py", line 440, in main
    logger.info('starting')
  File "/dtu/blackhole/1b/209339/miniconda3/envs/brain/lib/python3.11/logging/__init__.py", line 1849, in info
    self.log(INFO, msg, *args, **kwargs)
  File "/dtu/blackhole/1b/209339/miniconda3/envs/brain/lib/python3.11/site-packages/accelerate/logging.py", line 52, in log
    raise RuntimeError(
RuntimeError: You must initialize the accelerate state by calling either `PartialState()` or `Accelerator()` before using the logging utility.
